### DISCUSSION TITLE
Add back beatmap hitsounds toggle during replay playback

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#23838] [Q&A] [answered]
I think these used to be there, but were moved to the audio settings when local offset was introduced.
Sometimes i wanna jump into a map and listen to it with skin sounds and when i notice it doesn't work i remember i have to open game settings and toggle the option.

beatmap loading
![image](https://github.com/ppy/osu/assets/53081750/2f3e747e-b264-4f0a-8b9f-5cb05386a579)
replay playback
![image](https://github.com/ppy/osu/assets/53081750/e6939e32-d643-4f01-85f9-2410df099fda)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/23841
